cli.js

```diff
@@ -1542,6 +1579,15 @@ function prepCli(packageJson) {
     .option('-r, --retries <N>', 'set number of retries for each chunk before giving up (`infinite` for infinite)', 10)
     .option('-t, --meta-retries <N>', 'set number of retries for collating track feeds (`infinite` for infinite)', 5)
     .option('-d, --directory <DIR>', 'save tracks to DIR/..')
+    .option(
+      '-D, --check-dir <DIR>',
+      [
+        'check if tracks already exist in another DIR (repeatable)',
+        '(useful if you maintain multiple libraries)',
+        '(example: `-D dir1 -D dir2`)',
```

Comment:
Wonder if it makes sense to make this support comma-separated paths. Example: `-D dir1 -D dir2 -D dir3,dir4`

It's convenient. Leaving this as a comment for myself.

Reply:
Done